docs/getting-started/first-routing.md

```diff
@@ -121,7 +120,7 @@ kubectl expose deployment httpbin-1 --port=80
 ```
 
 ??? note "About the candidate"
-    In this tutorial, the image is the same as for the primary version. In a real world example, it would be different.
+    In this tutorial, image is the same as for the primary version. In a real world example, it would be different. The version label (`app.kubernetes.io/version`) can be used to distinguish between versions.
```

Comment:
```suggestion
    In this tutorial, the candidate image is the same as the one for the primary version. In a real world example, it would be different. The version label (`app.kubernetes.io/version`) can be used to distinguish between versions.
```